sirepo/template/raydata.py

```diff
@@ -78,7 +78,28 @@ def _paths():
 
 
 def background_percent_complete(report, run_dir, is_running):
-    return PKDict(percentComplete=0 if is_running else 100)
+    r = PKDict(percentComplete=0 if is_running else 100)
+    if report != 'pollBlueskyForScansAnimation':
+        return r
+    try:
+        t = float(pkio.read_text(_BLUESKY_POLL_TIME_FILE).strip())
+    except Exception as e:
+        if not pkio.exception_is_not_found(e):
+            raise
+        t = sirepo.simulation_db.read_json(
+            run_dir.join(template_common.INPUT_BASE_NAME),
+        ).models.pollBlueskyForScansAnimation.start
+
+    s = []
+    for k, v in catalog().search({'time': {'$gte': t}}).items():
+        t = max(t, v.metadata['start']['time'])
```

Comment:
v.metadata.start.time?

Reply:
v isn't a pkdict. It is a bluesky document